business/checkers/authorization/no_host_checker.go

```diff
@@ -63,7 +63,7 @@ func (n NoHostChecker) validateHost(ruleIdx int, to []*api_security_v1beta.Rule_
 		}
 
 		for hostIdx, h := range t.Operation.Hosts {
-			fqdn := kubernetes.GetHost(h, namespace, clusterName, n.Namespaces.GetNames())
+			fqdn := kubernetes.GetHost(h, namespace, "", n.Namespaces.GetNames())
```

Comment:
Can you change the method "kubernetes.GetHost" to not accept cluster name anymore?
As it by default will use the config one.

Reply:
Done in commit 6161aacbe9cce903d52a09b92b932359b753cd47